src/modules/registrypreview/RegistryPreviewUI/RegistryPreviewXAML/MainWindow.xaml

```diff
@@ -164,6 +176,9 @@
                         x:Uid="RegistryJumpToKeyButton"
                         Click="RegistryJumpToKeyButton_Click"
                         IsEnabled="True">
+                        <AppBarButton.Icon>
+                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xee49;"/>
```

Comment:
This one doesn't seem to be there on Windows 10 by default. Can we replace it with something else?

Reply:
Maybe E152 
<img width="64" alt="image" src="https://github.com/microsoft/PowerToys/assets/18704482/f191e0e5-f923-4e2d-860b-af88b3c1a702">

It's in both fonts but I'm not a huge fan of it.  I might just yeet it.

Reply:
I guess we could just remove it? Or use the same as used elsewhere? I think it's OK for two to have the same icon.